modules/javafx.graphics/src/main/java/com/sun/javafx/text/PrismTextLayout.java

```diff
@@ -1009,9 +1009,47 @@ private TextLine createLine(int start, int end, int startOffset) {
             leading = Math.max(leading, run.getLeading());
             length += run.getLength();
         }
+
+        /*
+         * Calculate the width of trailing spaces for the new TextLine so they're
+         * can be excluded when doing later alignment calculations:
```

Comment:
It reads not correct. Maybe like this?

```suggestion
         * Calculate the width of trailing spaces for the new TextLine so they
         * can be excluded when doing later alignment calculations:
```